modular_skyrat/modules/synth_construction/code/living.dm

```diff
@@ -0,0 +1,67 @@
+/mob/living/carbon/human/species/proc/create_bodyparts_body_only()
```

Comment:
instead of doing this, i'd add a crafting recipe to utilize a synth chassis item and a head you want to attach to it, so that the game isn't dealing with a literal /mob/living/carbon/human being printed straight from production machinery